cookbook/ui_customization/change_pim_locale.rst

```diff
@@ -0,0 +1,28 @@
+How to change the PIM locale
+============================
+
+In this cookbook, you will learn how to manage Akeneo PIM locales.
+
+How to allow more locales
+-------------------------
+
+The global Akeneo PIM locale and user interface locales are configurable. By default, the Akeneo PIM is configured to
+only allow locales translated to more than 80%.
+
+If you want to allow more locales to be accessible, you can configure this limit in
+``src/Pim/Bundle/LocalizationBundle/Resources/config/providers.yml`` file:
```

Comment:
nope. This file will be in vendors directory, integrators don't change these files.
You have to explain what parameter you need to override. e.g.

```    
If you want to allow more locales, you can change the limit by overriding the parameter:
parameters:
    pim_localization.provider.ui_locale.min_percentage: 0.7
```